sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_STORAGE_BLOBS_H
+#define AZ_STORAGE_BLOBS_H
+
+#include <az_contract.h>
+#include <az_http_pipeline.h>
+#include <az_http_response.h>
+#include <az_result.h>
+#include <az_span.h>
+#include <az_str.h>
+
```

Comment:
Could you please Ctrl+K-D (in VS) on all the new files to apply auto formatting?